tests/container_upgrade/container_upgrade_helper.py

```diff
@@ -121,17 +125,36 @@ def os_upgrade(duthost, localhost, tbinfo, image_url):
               "All critical services should be fully started!")
 
 
+def disable_features(duthost):
+    for service in service_list:
+        logger.info(f"Disabling {service} feature")
+        duthost.shell(f"config features state {service} disabled", module_ignore_errors=True)
+
+
+def enable_features(duthost):
+    for service in service_list:
+        logger.info(f"Enabling {service} feature")
+        duthost.shell(f"config features state {service} enabled", module_ignore_errors=True)
+
+
 def pull_run_dockers(duthost, creds, env):
     logger.info("Pulling docker images")
 
+    disable_features(duthost)
     registry = load_docker_registry_info(duthost, creds)
     for container, version, name in zip(env.containers, env.container_versions, env.container_names):
         docker_image = f"{registry.host}/{container}:{version}"
         download_image(duthost, registry, container, version)
+        # Tag the image to replace the old image
```

Comment:
># Tag the image to replace the old image [](http://example.com/codeflow?start=8&length=40)

Is it better do not tag latest, but use right version during `docker run`?

Reply:
The gnmi service will run docker with the latest tag, if we do not tag the image, gnmi docker will be replaced by gnmi service.

Reply:
I have removed the docker tag command